epub33/core/index.html

```diff
@@ -5425,7 +5425,7 @@ <h4>Orientation</h4>
 						</dd>
 					</dl>
 
-					<p>EPUB Creators MUST NOT declare the <code>rendition:orientation</code> property more than
+					<p data-tests="fxl-orientation_duplication">EPUB Creators MUST NOT declare the <code>rendition:orientation</code> property more than
```

Comment:
I recommend adding an anchor every time you add data-tests.

At minimum, you'll need it to link from the test back to the spec (the isReferencedBy meta tag), but it's just handy to have more anchors, too :)